src/client/interpreter/locators/services/condaEnvFileService.ts

```diff
@@ -0,0 +1,69 @@
+"use strict";
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import { IInterpreterVersionService } from '../../interpreterVersion';
+import { IInterpreterLocatorService, PythonInterpreter } from '../../contracts';
+import { AnacondaDisplayName, AnacondaCompanyName, CONDA_RELATIVE_PY_PATH } from './conda';
+
+/**
+ * The command 'conda info --envs' doesn't seem to return all environments
+ * Environments created using the command 'conda create --p python=x.x' are not returned by the above command
+ * However all these environments seem to be listed in the environments.txt file (confirmed on windows and linux)
+ * @export
+ * @class CondaEnvFileProvider
+ * @implements {IInterpreterLocatorService}
```

Comment:
What are these comments for? They seem to (be trying to) mirror what the code already says so I'm wondering what the benefit is.

Reply:
I thought it would be best to write the comments out. Else one might ask whey use environments.txt as well as ```conda info --env```. This makes it obvious that environments.txt can contain additional items. Comment removed.

Reply:
Sorry for the ambiguity of that comment, I meant the `@export` comments and such more than the block comments. (I totally support block comments 😄 )